.github/workflows/github-issue-stats.yml

```diff
@@ -0,0 +1,53 @@
+name: Generate GitHub Issue Statistics
+
+on:
+  workflow_dispatch:
+    inputs:
```

Comment:
Never seen this in my life - how can this be used? Thats cool! When I trigger the workflow on GitHub, is it basically asking me for these inputs?

Reply:
yes, that is how it should work. this final test of the workflow can only be done once the PR is on `master` since it's a new workflow!